ostd/src/mm/page_table/cursor.rs

```diff
@@ -141,7 +141,7 @@ where
     /// Note that this function does not ensure exclusive access to the claimed
     /// virtual address range. The accesses using this cursor may block or fail.
     pub fn new(pt: &'a PageTable<M, E, C>, va: &Range<Vaddr>) -> Result<Self, PageTableError> {
-        if !M::covers(va) {
+        if !M::covers(va) || va.end == 0 {
```

Comment:
It think that `va.is_empty()` covers more bad cases here, for example, `start >= end`, which we don't handle well.

Reply:
updated